source/locationHelper.py

```diff
@@ -6,16 +6,41 @@
 
 """Classes and helper functions for working with rectangles and coordinates."""
 
-from collections import namedtuple
+from collections import namedtuple, Sequence
 import windowUtils
 import winUser
-from ctypes.wintypes import RECT, SMALL_RECT, POINT
+from ctypes.wintypes import RECT, POINT, DWORD
 import textInfos
 import wx
 
 class Point(namedtuple("Point",("x","y"))):
 	"""Represents a point on the screen."""
 
+	@classmethod
+	def fromNonInts(cls, *nonInts):

```

Comment:
Does this work with anything other than a float? I'm concerned that the types that can be used here are are not so obvious.  If it is only going to work on floats (and by looks of it also ints) then a name like `fromFloatCollection` might be better?

Edit:
Upon reading the tests (yay tests), I have realised this is also intended to work with strings. I think this might be a mistake. Firstly, is there anywhere that actually returns coordinates in a string? Second, this would make it all too easy for someone to forget to consider variations in the format of the string due to localisation.

Reply:
Somehow, It must have been late when I believed that it would make sense to create a rectangle from numbers that are saved in strings, but I guess that doesn't make very much sense. I applied your suggestion.